packages/launch/src/focused-launch/success/index.tsx

```diff
@@ -23,16 +23,16 @@ import './style.scss';
 // Success is shown when the site is launched but also while the site is still launching.
 // This view is technically going to be the selected view in the modal even while the user goes through the checkout flow (which is rendered on top of this view).
 const Success: React.FunctionComponent = () => {
-	const { redirectTo, siteId } = React.useContext( LaunchContext );
+	const { redirectTo, siteId, getCurrentLaunchFlowUrl } = React.useContext( LaunchContext );
 
 	const isSiteLaunching = useSelect( ( select ) => select( SITE_STORE ).isSiteLaunching( siteId ) );
 
-	const {
-		unsetModalDismissible,
-		hideModalTitle,
-		closeFocusedLaunch,
-		disablePersistentSuccessView,
-	} = useDispatch( LAUNCH_STORE );
+	const isSelectedPlanPaid = useSelect(
+		( select ) => !! select( LAUNCH_STORE ).getPaidPlanProductId(),
+		[]
+	);
```

Comment:
Should be refactored once #49930 is merged (see [this comment](https://github.com/Automattic/wp-calypso/pull/49930/files#r575511915) for context)

Reply:
Cool. Let's test it again when #49930 is deployed.

Reply:
As discussed in Resolving this comment, as it will be fixed in #49930 (see https://github.com/Automattic/wp-calypso/pull/49930#issuecomment-779856916)